packages/amplify-category-analytics/src/lib/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const CategoryName = 'analytics';
```

Comment:
I'm not sure if it makes sense to change in this PR, but we don't want `src` files in a `lib` folder

Reply:
I feel that this is a small enough change that we can take as part of this refactor